hw2/index.js

```diff
@@ -0,0 +1,44 @@
+function NamedOne (firstName, lastName) {
+
+    // this.lastName = lastName;
+    // this.fullName = `${this.firstName} ${this.lastName}`;
+
+    let res = {
+
+        firstName: lastName,
+        lastName: lastName,
+
+        get fullName() {
+            return `${this.firstName} ${this.lastName}`;
+        },
+
+        set fullName(val) {
+            let nameArr = val.split(' ');
+
+            if (nameArr.length !== 2) {
+                return (console.log(`Your name isn't correct`));
+            } else {
+                this.firstName = nameArr[0];
+                this.lastName = nameArr[1];
+            }
+        }
+    }
+    return res;
+}
+
+let namedOne = new NamedOne ("Naomi", "Wang");
+
+console.log('fullName: ', namedOne.fullName);
+
```

Comment:
@matveyprupas get fullName returns wrong value - "Wang Wang" instead of "Naomi Wang", Please, fix it.

Reply:
Done